src/xdocs/config_filters.xml

```diff
@@ -889,6 +889,19 @@ public class UserService {
           all methods with name <i>sayHelloWorld</i> inside <i>FileOne</i>
           and <i>FileTwo</i> files:
         </p>
+        <p>
+        Currently, xpath queries support one type of attribute <code>@text</code>.
+        <code>@text</code> - addresses to the value of the node. For example: variable value,
+        annotation value, text content and etc.
+        Only the following token types support <code>@text</code> attribute:
+        <code>TokenTypes.IDENT</code>, <code>TokenTypes.STRING_LITERAL</code>,
+        <code>TokenTypes.CHAR_LITERAL</code>, <code>TokenTypes.NUM_LONG</code>,
+        <code>TokenTypes.NUM_INT</code>, <code>TokenTypes.NUM_DOUBLE</code>,
+        <code>TokenTypes.NUM_FLOAT</code>.
+        These token types were selected because only their values are useful and can be used
```

Comment:
"These token types were selected because only their text values are different in content from token type and represent text value from file and can be used"

Reply:
done